# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/nsWindow.cpp
# Commit: 18f23fa58771
# Full Hash: 18f23fa5877125926953f6d53a9e2c881d0b9be2
# Author: Florian Queze <florian@queze.net>
# Date: 2022-11-14 21:44:03
# Regressor Bug: 1671490
# File Overlap Count: 1
# Description:
#   Bug 1671490 - Mark the GPU process as being in the foreground whenever the parent process is in the foreground to make their priorities (and priority boosts) match, r=bas.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161065
# ==============================================================================

diff -r c1fef68a839f -r 18f23fa58771 widget/windows/nsWindow.cpp
--- a/widget/windows/nsWindow.cpp	Mon Nov 14 11:03:18 2022 -0500
+++ b/widget/windows/nsWindow.cpp	Mon Nov 14 16:05:34 2022 +0000
@@ -6194,6 +6194,10 @@
       }
     } break;
 
+    case WM_ACTIVATEAPP: {
+      GPUProcessManager::Get()->SetAppInForeground(wParam);
+    } break;
+
     case WM_MOUSEACTIVATE:
       // A popup with a parent owner should not be activated when clicked but
       // should still allow the mouse event to be fired, so the return value
